esmvaltool/cmor/_fixes/CMIP6/BCC_ESM1.py

```diff
@@ -0,0 +1,65 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for BCC-ESM1."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Common fixes to all vars"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes error in time coordinate, sometimes contains trailing zeros
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            try:
+                old_time = cube.coord('time')
+                if old_time.is_monotonic():
```

Comment:
this does nothing, can be removed

Reply:
Only some of the files from BCC-ESM1 have a problem with the time coordinate. This statement makes sure that files that are OK are not modified, so the statement in question cannot be removed.

Reply:
In that case, did you mean to use `continue` instead of `pass`? Because currently this code really does nothing.